.github/workflows/tests_lkcuda_python.yml

```diff
@@ -275,10 +275,12 @@ jobs:
           OMP_PROC_BIND: false
         run: |
           cd main/
-          # for device in lightning.qubit lightning.kokkos; do
-          for device in lightning.qubit; do
-            pl-device-test --device ${device} --skip-ops --shots=20000 $COVERAGE_FLAGS --cov-append
-            pl-device-test --device ${device} --shots=None --skip-ops $COVERAGE_FLAGS --cov-append
+          for device in lightning.qubit lightning.kokkos; do
+            echo "FDX test01: ${device} "
```

Comment:
You may want to remove that, right?
```suggestion
```

Reply:
Yes, sorry I forgot to remove all these prints :worried: